### ISSUE TITLE
Support request: Display some icon when loading shapefiles through worker?

### ISSUE THREAD

**Issue by sgofferj** [calvinmetcalf/leaflet.shapefile#59]
JS beginner here. I'm tinkering with your plugin and I was wondering if there is some relatively easy way to display a message or an icon indicating that the shapefile is currently loading while the worker is loading it.

**Comment by calvinmetcalf** [comment]
Yes there is a leaflet spin plugin that this should work with, I think it's
mentioned in the readme
